lib/OpenQA/Test/PatchDeparse.pm

```diff
@@ -15,7 +15,8 @@ if (
   )
 {
 
-#<<<  do not let perltidy touch this
+#<<<  do not let perltidy nor perlcritic touch this
+## no critic (TestingAndDebugging::ProhibitNoStrict ValuesAndExpressions::ProhibitInterpolationOfLiterals)
 # This is not our code, and formatting should stay the same for
 # better comparison with new versions of B::Deparse
```

Comment:
Apparently the link to the [discussion in #30](https://github.com/os-autoinst/os-autoinst-common/pull/32#pullrequestreview-1781943388) doesn't lead to the resolved thread anymore, I repeat it here:
```
# This is not our code, and formatting should stay the same for
￼# better comparison with new versions of B::Deparse		￼# better comparison with new versions of B::Deparse
```
I don't understand why it is so difficult to leave this code alone.
Can't you just add `## no critic`?
Why is it so important for you to tidy this part of the code?

Reply:
>  Why is it so important for you to tidy this part of the code?

I'm very adamant on minimizing exceptions for linting. I *still strongly* believe that the whitespace changes are irrelevant for the actual patch. But I'll proceed to revert them and mark the file for ignoring.

Reply:
Updated in [c045e31](https://github.com/os-autoinst/os-autoinst-common/commit/c045e315cc5d8d0c361e4672564fedc3d306d263)

Reply:
thanks :)